internal/controller/client_proxy.go

```diff
@@ -100,13 +100,6 @@ func (k *controllerProxy) ListResources(labels map[string]string, namespaces ...
 				{Kind: "CustomResourceDefinition"},
 			},
 		},
-		{
-			GroupVersion: "cert-manager.io/v1",
-			APIResources: []metav1.APIResource{
-				{Kind: "Certificate", Namespaced: true},
-				{Kind: "Issuer", Namespaced: true},
-			},
-		},
```

Comment:
why are we removing this?

Reply:
There are 2 reasons for that:
1. In clusterctl they explicitly [declare](https://github.com/kubernetes-sigs/cluster-api/blob/6dc0819f7dc07189ae1a30f2d4be0c982db6bbd6/cmd/clusterctl/client/cluster/proxy.go#L67-L68) that this function should not return cert-manager objects. It's better to keep things consistent with cluster-api.
1. After upgrading to controller-runtime v0.15.0 some tests fail because they need cert-manager CRDs installed (I still don't understand why it works in the previous versions). I don't want to add cert-manager to the operator dependencies, so I just decided to remove the section.